language/Turkish/strings.po

```diff
@@ -1980,6 +2034,12 @@ msgid ""
 "If this is your first time using %2$s, please see the %2$s.pdf file in the "
 "docs folder for information on setting up and using %2$s."
 msgstr ""
+"Sistem ROM'u bulunamadı. Lütfen %1$s.ini 'de belirtilen rompath ayarını "
+"kontrol edin.\n"
+"\n"
+"%2$s'yi ilk kez kullanıyorsanız, lütfen %2$s'yi kurma ve kullanma hakkında "
+"bilgi için dokümanlar klasöründeki %2$s.pdf dosyasına bakın."
```

Comment:
Oh, one other thing.  In this message, “docs” is the name of the folder containing the documentation when you download and extract MAME.  The word “docs” should be preserved literally in the translated message, as the name of the folder will not be localised.

Reply:
fixed it

Reply:
also thanks from me :)
i'll try to finish the rest in month...